app/controllers/items_controller.rb

```diff
@@ -120,7 +157,11 @@ def workflow_view
       format.any(:png, :svg, :jpeg) do
         graph = @workflow.graph
         fail ActionController::RoutingError.new('Not Found') if graph.nil?
-        send_data graph.output(request.format.to_sym => String), :type => request.format.to_s, :disposition => 'inline'
+        send_data(
```

Comment:
Similar to other comments related to necessity for the issue at hand. I'm all for conformity to a style, just not at the expense of confusing a future reviewer in a single commit or PR.